recipes/zxing-cpp/recipe.yaml

```
@@ -0,0 +1,57 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: zxing-cpp
+  version: "2.3.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/zxing_cpp-${{ version }}.tar.gz
+  sha256: 3babedb67a4c15c9de2c2b4c42d70af83a6c85780c1b2d9803ac64c6ae69f14e
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+
+  build:
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+        - pybind11-global <3.0.0
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - ${{ stdlib('c') }}
+  host:
+    - python
+    - setuptools >=42,<81.0.0
+    - wheel
+    - cmake >=3.15,<4.0.0
+    - make
+    - pkg-config
+    - pybind11-global <3.0.0
+    - pip
```

```suggestion
  build:
    - if: build_platform != target_platform
      then:
        - python
        - cross-python_${{ target_platform }}
        - pybind11-global <3.0.0
    - ${{ compiler("c") }}
    - ${{ compiler("cxx") }}
    - ${{ stdlib('c') }}
    - cmake >=3.15,<4.0.0
    - make
    - pkg-config
  host:
    - python
    - setuptools >=42,<81.0.0
    - wheel
    - pybind11-global <3.0.0
    - pip
```
Sorry I missed this in the first round of review, but these dependencies need to be in build, not host, or the cross-compilation will fail.